lib/gds_api/mapit.rb

```diff
@@ -12,6 +12,10 @@ def areas_for_type(type)
     get_json("#{base_url}/areas/#{type}.json")
   end
 
+  def area_for_code_type(code_type, code)
```

Comment:
On naming, this is really getting the area for a code - `area_for_code_type` sounds as though it might get all areas for a code type as `areas_for_type` does, instead of the single area with the code of a given type. I think I'd find `area_for_code` clearer - what do you think?